config/i18n-tasks.yml

```diff
@@ -110,6 +110,10 @@ ignore_unused:
   - 'user_mailer.email_confirmation_instructions.subject'
   - 'valid_email.validations.email.invalid'
   - 'errors.attributes.*'
+  -  'two_factor_authentication.two_factor_choice_options.configurations_added'
```

Comment:
I'd have a soft-preference to either using the [inline comment hint](https://github.com/glebm/i18n-tasks#fine-tuning) or just avoid the dynamic key, since it'd be easier to maintain over time closer to the source.

Reply:
I refactored this since the method in all 6 files was going to get long and put it into the presenter, with a mfa_config_count method that I could override in each individual presenter since that was the only real difference. Backup codes needed to be separate, but this is cleaner overall.